api/src/services/applications/get_application.py

```diff
@@ -23,6 +28,15 @@ def get_application(db_session: db.Session, application_id: UUID, user: User) ->
             selectinload(Application.application_users)
             .selectinload(ApplicationUser.user)
             .selectinload(User.linked_login_gov_external_user),
+            selectinload(Application.competition)
+            .selectinload(Competition.competition_forms)
+            .selectinload(CompetitionForm.form),
+            selectinload(Application.competition).selectinload(
+                Competition.opportunity_assistance_listing
+            ),
+            selectinload(Application.competition).selectinload(
+                Competition.link_competition_open_to_applicant
```

Comment:
I wonder if at this point it is better to just do a `selectinload("*")` and then exclude the ones we don't want.

We wouldn't want some of the stuff on the user table (saved searches/opportunities), we wouldn't want the competition.opportunity or competition.applications - but I think that's all we'd need to exclude? Worth seeing if that's any clearer, but no need to look into that too much.

FYI - I just merged the PR where the application forms point to the competition forms table, so this might need to be adjusted slightly with that pulled in.